### PR TITLE
rename non-production auth cookies

### PR DESCRIPTION
**Pull request by sanyogita14**
Authentication cookies renamed as `rds-session-<NODE_ENV>` for non-production environments

### CODE REVIEW

File: config/default.js [hunk 1]
```diff
@@ -5,6 +5,8 @@
  *
  * Documentation: https://github.com/lorenwest/node-config/wiki/Configuration-Files
  */
+
+var NODE_ENV = process.env.NODE_ENV
```
Comment (swarajpure) [inline config/default.js:9]: Can we please use `const` here instead of `var` since the value will not be changed?

**Review by swarajpure — CHANGES_REQUESTED** [review]
One minor change in the config file required.

Also, the tests are failing and they're failing because the cookie used to authenticate the user is being set by the name `rds-session` and according to the implementation of this PR, the cookie name will be called `rds-session-test`.

Since the tests are looking for a cookie named `rds-session`, they wont be able to find it and hence, the tests are failing. So you might want to declare a variable `cookie` or `cookieName` in all the files where we are using the cookie and let it take the cookie name from the config and pass that variable wherever we are passing the cookie in the request. You can try doing a global search for ```.set('cookie', `rds-session=${jwt}\`)``` in VSCode and you'll understand what I want to say.

Thank you! 😄

File: config/default.js (hunk 1 above)
Comment (sanyogita14) [reply config/default.js]: Yes. Will carry out the changes. Thanks for the help!

File: config/default.js [hunk 2]
```diff
@@ -38,7 +40,7 @@ module.exports = {
   },
 
   userToken: {
-    cookieName: 'rds-session',
+    cookieName: `rds-session-${NODE_ENV}`,
```
Comment (ankurnarkhede) [inline config/default.js:43]: We just need to rename the non-production cookies.

File: test/integration/healthController.test.js [hunk 3]
```diff
@@ -1,12 +1,14 @@
 const chai = require('chai')
 const { expect } = chai
 const chaiHttp = require('chai-http')
+// const { config } = require('winston')
```
Comment (ankurnarkhede) [inline test/integration/healthController.test.js:4]: Why is this line added?

File: test/integration/healthController.test.js [hunk 4]
```diff
@@ -1,12 +1,14 @@
...
 chai.use(chaiHttp)
 
 const app = require('../../server')
 const authService = require('../../services/authService')
 const addUser = require('../utils/addUser')
-
+const config = require('config')
```
Comment (ankurnarkhede) [inline test/integration/healthController.test.js:10]: `config` is accessible globally. Can you check if its accessible in the test scripts?
If yes, we can remove the import for it.

File: config/default.js (hunk 2 above)
Comment (ankurnarkhede) [reply config/default.js:43]: @sanyogita14 , can you override the cookie name in production config to be `rds-session`?
We do not want the cookie name in production to be `rds-session-production`.

**Review by ankurnarkhede — CHANGES_REQUESTED** [review]

File: config/default.js (hunk 2 above)
Comment (sanyogita14) [reply config/default.js:43]: Alright..so I'll just specify it separately for the production file, right?

File: test/integration/healthController.test.js (hunk 4 above)
Comment (sanyogita14) [reply test/integration/healthController.test.js:10]: Yes I tried but the tests were failing so I had to import it into all the files

File: config/default.js (hunk 2 above)
Comment (ankurnarkhede) [reply config/default.js:43]: Yes @sanyogita14

**Review by ankurnarkhede — APPROVED** [review]
